### PR TITLE
[opentelemetry-operator] Improve Operator deployment args templating

### PR DESCRIPTION
**Pull request by erichsueh3**
Addresses #526.

### CODE REVIEW

**Review by Allex1 — APPROVED** [review]

**Review by TylerHelmuth — COMMENTED** [review]
@erichsueh3 thanks for adding this capability.  I just merged in a different operator PR so this PR needs a rebase/new version bump/example re-generation.

**Comment by erichsueh3** [comment]
> @erichsueh3 thanks for adding this capability. I just merged in a different operator PR so this PR needs a rebase/new version bump/example re-generation.

Rebased and bumped to 0.19.1

File: charts/opentelemetry-operator/values.yaml [hunk 1]
```diff
@@ -62,9 +62,27 @@ manager:
...
+      tag: ""
+    nodejs:
+      repository: ""
+      tag: ""
+    python:
+      repository: ""
+      tag: ""
+    dotnet:
+      repository: ""
+      tag: ""
```
Comment (dmitryax) [inline charts/opentelemetry-operator/values.yaml:81]: Let's put it closer to the `collectorImage` config? Does specifying `autoInstrumentationImage` _enable_ the auto instrumentation or needed just to override default images?

Comment (erichsueh3) [reply charts/opentelemetry-operator/values.yaml]: Moved to under `collectorImage` config, and it should just be overriding the default images

**Review by TylerHelmuth — APPROVED** [review]

**Comment by erichsueh3** [comment]
Do we need to re run the CI on this? Looks like it's hanging

**Review by dmitryax — APPROVED** [review]

**Pull request merged**